omero/sysadmins/unix/install-web.txt

```diff
@@ -55,6 +56,9 @@ customization options.
     install-web/install-apache
     install-web/install-nginx
 
+.. note:: Apache is deprecated as of OMERO 5.2.6 and will not be supported for
+    5.3.
+
```

Comment:
Sorry for a slow response on the doc, I would suggest to add note:

```
If Apache HTTP Server is the only way to provide Internet access to internal clients that are otherwise restricted by a firewall. You should configure Apache HTTP Server as a forward proxy to Nginx, for more details see https://httpd.apache.org/docs/current/mod/mod_proxy.html
```